_posts/en/newsletters/2021-04-14-newsletter.md

```diff
@@ -0,0 +1,225 @@
+
+The review club discussion focused on understanding the change and its potential
+benefits.
+
+{% include functions/details-list.md
+  q0="What are the advantages of a [BIP90][] buried deployment over a [BIP9][]
+     version bits deployment?"
+  a0="A buried deployment simplifies the deployment logic by replacing the test
+     that governs enforcement with simple height checks, thereby reducing the
+     technical debt associated with deployment of those consensus changes."
+  a0link="https://bitcoincore.reviews/19438#l-132"
+
+  q1="How many buried deployments are defined by this PR?"
+  a1="Five: height in coinbase, CLTV ([BIP65][] `CHECKLOCKTIMEVERIFY`), strict
+     DER signatures, CSV ([BIP112][] `OP_CHECKSEQUENCEVERIFY`), and segwit. They
+     are listed in the `BuriedDeployment` enumerator proposed by the PR in
+     [src/consensus/params.h#L14-22](https://github.com/bitcoin/bitcoin/blob/e72e062e/src/consensus/params.h#L14-L22).
+     One could argue that the [Satoshi-era
+     softforks](/en/topics/soft-fork-activation/#2009-hardcoded-height-consensus-nlocktime-enforcement)
+     are also buried."
+  a1link="https://bitcoincore.reviews/19438#l-75"
+
+  q2="How many version bits deployments are defined by this PR?"
+  a2="Two: testdummy and schnorr/taproot (BIPs 340-342), listed by the PR in
+     [src/consensus/params.h#L25-31](https://github.com/bitcoin/bitcoin/blob/e72e062e/src/consensus/params.h#L25-L31)."
+  a2link="https://bitcoincore.reviews/19438#l-96"
+
+  q3="If the taproot soft fork is activated and we later want to bury that
+     activation method, what changes would need to be made to Bitcoin Core, if
+     this PR is merged?"
+  a3="The main change would be greatly simplified compared to the current code:
+     move the `DEPLOYMENT_TAPROOT` line from the `DeploymentPos` enum to the
+     `BuriedDeployment` one. Most importantly, [no validation logic would need
+     to be changed][burying taproot]."
+  a3link="https://bitcoincore.reviews/19438#l-227"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding (none yet; will update or remove the section Tuesday)
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21594][] adds a network field to RPC `getnodeaddresses`
+  to help identify nodes on various networks (i.e. ipv4, ipv6, I2P, onion).
+  The author has also proposed that this sets the groundwork for a future
```

Comment:
maybe s/sets/set/ or s/sets/lay/ (laying the groundwork) or could just "that a follow-up patch enable..."

Reply:
I also hadn't heard "set the groundwork" before, but a google search revealed that it's about as popular as "lay the groundwork", so I didn't comment. I agree that "lay the groundwork" is slightly better.

Reply:
> I also hadn't heard "set the groundwork" before, but a google search revealed that it's about as popular as "lay the groundwork"

Thanks! TIL